modules/player/src/test/java/unnamed/mmo/player/damage/TestDamageIntegration.java

```diff
@@ -0,0 +1,53 @@
+package unnamed.mmo.player.damage;
+
+import net.minestom.server.MinecraftServer;
+import net.minestom.server.attribute.Attribute;
+import net.minestom.server.attribute.AttributeOperation;
+import net.minestom.server.entity.EntityType;
+import net.minestom.server.entity.LivingEntity;
+import net.minestom.server.item.ItemStack;
+import net.minestom.server.item.Material;
+import net.minestom.server.item.attribute.AttributeSlot;
+import net.minestom.server.item.attribute.ItemAttribute;
+import org.junit.jupiter.api.Test;
+import unnamed.mmo.damage.DamageProcessor;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class TestDamageIntegration {
+
+    @Test
+    public void testCanApplyDamage() {
+        // Required for not NullPointering when applying damage because of an event call
```

Comment:
I copied `EventUtil#safeDispatch` from the quest branch into main, that would be preferrable anywhere you dispatch an event (just noops if MinecraftServer has not been initialized)

Reply:
Unfortunately that won't help a whole lot, as the call to the event is part of the Entity.damage(); function inside Minestom.

Reply:
Unfortunate.